sample/golang-echo/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.9-alpine
```

Comment:
https://hub.docker.com/r/library/golang/tags/ only lists the alpine3.6 versions as good. And I misunderstood what you meant by LTS. I thought you were talking about which version of alpine has best support.